couchbase_v3/couchbase.php

```diff
@@ -1516,7 +1532,7 @@ public function setName(string $name): CollectionSpec {}
 
     public function setScopeName(string $name): CollectionSpec {}
 
-    public function setMaxExpiry(float $ms): CollectionSpec {}
+    public function setMaxExpiry(long $ms): CollectionSpec {}
```

Comment:
This is incorrect change as there is no such typehint

Reply:
fixed